docs/src/main/sphinx/release/release-479.md

```diff
@@ -0,0 +1,194 @@
+
+* Replace `join-pushdown.with-expressions` configuration property with `deprecated.join-pushdown.with-expressions`. The new property will be removed in a future version.
+
+## Faker connector
+
+## Google Sheets connector
+
+## Hive connector
+
+* Remove `s3.exclusive-create` config. ({issue}`27372`)
+* Fix updating table statistics when running INSERT queries with OVERWRITE behaviour. ({issue}`26517`)
+* Harden hierarchical namespace check in Azure with root blob fallback check. ({issue}`27278`)
+* Avoid reading unusually large pages from parquet files. ({issue}`27148`)
+* Fix potential `FileAlreadyExistsException` failure when network failures during s3 write. ({issue}`27330`)
+
+## Hudi connector
+
+* Add `APPLICATION_DEFAULT` authentication type for GCS. ({issue}`26984`)
+* Remove unauthentication access support in `SERVICE_ACCOUNT`. ({issue}`26984`)
+* Harden hierarchical namespace check in Azure with root blob fallback check. ({issue}`27278`)
+* Avoid reading unusually large pages from parquet files. ({issue}`27148`)
+* Fix potential `FileAlreadyExistsException` failure when network failures during s3 write. ({issue}`27330`)
+* Fix failure when listing empty directories on HDFS. ({issue}`26897`)
+
+## Iceberg connector
+
+* Add support for disabling token exchange via the `iceberg.rest-catalog.oauth2.token-exchange-enabled` config property. ({issue}`27174`)
+* Add `retain_last` and `clean_expired_metadata` options to `expire_snapshots` command. ({issue}`27357`)
+* Remove `s3.exclusive-create` config. ({issue}`27372`)
+* Add `APPLICATION_DEFAULT` authentication type for GCS. ({issue}`26984`)
+* Remove unauthentication access support in `SERVICE_ACCOUNT`. ({issue}`26984`)
+* Fix failures when querying `$files` table after changes to table partitioning. ({issue}`26746`)
+* Fix incorrect results due to incorrect pushdown of `IS NOT DISTINCT FROM`. ({issue}`27213`)
+* Harden hierarchical namespace check in Azure with root blob fallback check. ({issue}`27278`)
+* Avoid reading unusually large pages from parquet files. ({issue}`27148`)
+* Fix potential `FileAlreadyExistsException` failure when network failures during s3 write. ({issue}`27330`)
+* Fix potential failure when dropping a schema with cascade. ({issue}`27361`)
+
+## Ignite connector
+
+* Replace `join-pushdown.with-expressions` configuration property with `deprecated.join-pushdown.with-expressions`. The new property will be removed in a future version.
+
+## JMX connector
+
+## Kafka connector
+
+## Loki connector
+
+* Fix failure when initializing the connector. ({issue}`27180`)
+
+## MariaDB connector
+
+* Replace `join-pushdown.with-expressions` configuration property with `deprecated.join-pushdown.with-expressions`. The new property will be removed in a future version.
+
+## Memory connector
+
+* Add support for setting and dropping column defaults via `ALTER TABLE ... ALTER COLUMN`. ({issue}`26162`)
+* Fix concurrent modification exception in `RENAME SCHEMA`. ({issue}`27205`)
```

Comment:
What's "concurrent modification exception"? When does it happen? Describe those conditions, instead.

Reply:
applied